internal/api/server.go

```diff
@@ -136,12 +144,14 @@ func NewServer(
 	cfg config.ServerConfig,
 	kubeClient kubernetes.Client,
 	internalClient client.Client,
+	rolloutsEnabled bool,
```

Comment:
I suspect longer term we will find more things we need to pass in and might opt for some kind of config object here, but I think this is suitable for now.

Reply:
Yes, or `ServerConfig` being utilized better. As I am not sure what to think of passing config _and_ options.

Reply:
🤦‍♂️ I overlooked the `ServerConfig` that was already there... I might prefer if this bool were a field in there instead of loose.

Reply:
I'll do a quick follow up tomorrow as I initially didn't want to touch it due to it dealing with loading user defined settings.

But I think a way I do feel comfortable with is to overwrite / downgrade these settings if they are not feasible (due to e.g. the CRDs not being present).

Reply:
It's probably possible though for the CRDs to be present on the cluster and the integration is still not desired.